### PR TITLE
Bug 1136346 - 'Custom location' header is not consistent+autolander

### PR DESCRIPTION
**Pull request by martapiekarska**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[martasect martasect](https://github.com/martasect) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9e9159c8e2a4)

File: apps/privacy-panel/locales/privacypanel.en-US.properties [hunk 1]
```diff
@@ -52,7 +52,7 @@ exceptions-application-description = Adjust the custom location accuracy level o
 
 
 # ALA-10 (ALA: Define Custom Location)
-custom-location-ala-10 = Custom location
+Custom-Location-ala-10 = Custom Location
```
Comment (crh0716) [inline apps/privacy-panel/locales/privacypanel.en-US.properties]: In this case it is not required to change the string id.

**Comment by try-server-hook** [comment]
[martasect martasect](https://github.com/martasect) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=91a54392c736)

**Comment by try-server-hook** [comment]
[martasect martasect](https://github.com/martasect) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b337d32b0ae7)

**Pull request merged**